Example/Sources/Resources/Assets.xcassets/camera.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
```

Comment:
Can we either use PDF file in vectors, or use SF symbols (for iOS 13+) and system icon (https://developer.apple.com/documentation/uikit/uiapplicationshortcuticon/icontype/capturephoto for iOS 12)? (or https://developer.apple.com/documentation/uikit/uibarbuttonitem/systemitem/camera) - check https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/system-icons/

Reply:
@Kaspik 
I've changed to SF Symbols so I made a condition to check if the app run on ios 13 then use System-icon

Reply:
Can we use `UIApplicationShortcutIconTypeCapturePhoto` for iOS 12 and remove this asset completely?